compiler/crates/relay-typegen/src/visit.rs

```diff
@@ -1566,8 +1573,8 @@ fn make_prop(
                                          });
                                          let assignable_fragment_spread_ref= Prop::KeyValuePair(KeyValuePairProp {
```

Comment:
Unrelated: Strange that `cargo fmt` is not adding a space between `assignable_fragment_spread_ref` and `=`...

Reply:
Was `cargo fmt` not run in a while? If I run it locally I get ~150 file changes. But I think my Rust toolchain is just setup incorrectly...